config/config.go

```diff
@@ -57,6 +64,33 @@ func NewLocal(cfgPath, basepath string) (*Local, error) {
 		return nil, fmt.Errorf("failed to unmarshal local configuration (%s): %w", cfgPath, err)
 	}
 
+	// Check that origin matches issue tracker's format
+	var validOrigin bool
+	switch cfg.IssueTracker {
+	case IssueTrackerJira:
+		validOrigin = jiraOriginPattern.MatchString(cfg.Origin)
+		break
+	case IssueTrackerGithub:
+		validOrigin = githubOriginPattern.MatchString(cfg.Origin)
+		break
+	case IssueTrackerGitlab:
+		validOrigin = gitlabOriginPattern.MatchString(cfg.Origin)
+		break
+	case IssueTrackerPivotal:
+		validOrigin = pivotalOriginPattern.MatchString(cfg.Origin)
+		break
+	case IssueTrackerRedmine:
+		validOrigin = redmineOriginPattern.MatchString(cfg.Origin)
+		break
+	case IssueTrackerInvalid:
```

Comment:
Could you convert this switch-case into a map - `map[IssueTracker]*Regexp`.

Afterwards, validation can look like:
```
pattern, !ok := originPatterns[cfg.IssueTracker]
if !ok {
   // return err
}

// validate pattern matches
```

Reply:
Also, please make sure this map is close to the enum declaration to make things easier for future maintainers once a new integration is added.

Reply:
resolved in eaf4e4a